tests/external-links.js

```diff
@@ -269,9 +269,11 @@ async function updateGithubIssue(urlResults) {
 
     try {
       const lines = body.split(/\r?\n/); // support both \n and \r\n newline types
-      const firstContentLine = lines.indexOf(`<table>`) + 2;
-      lines.splice(0, firstContentLine); // delete first lines which only hold general data
       for (const line of lines) {
+        if (!line.startsWith(`<tr><td nowrap>`) {
```

Comment:
Just missing a bracket, then it runs fine.
```suggestion
        if (!line.startsWith(`<tr><td nowrap>`)) {
```